### PR TITLE
add win32 build rules

### PR DESCRIPTION
**Pull request by estebanlm**

### CODE REVIEW

File: build.win32x86/common/Makefile [hunk 1]
```diff
@@ -250,24 +260,25 @@ $(VMDEF) $(VMEXP) $(VMLIB): $(VMOBJ)
 .PHONY: $(OBJDIR)/%.lib $(OBJDIR)/%.dll
 
 # Check for Makefile in win32 plugins directory otherwise use default Makefile
-plugin-makefile = $(firstword $(realpath \
-					$(PLATDIR)/win32/plugins/$(*F)/Makefile.plugin \
-					../common/Makefile.plugin))
+plugin-makefile = $(realpath $(firstword $(wildcard $(WIN32PLUGINSDIR)/$(1)/Makefile.plugin ../common/Makefile.plugin)))
```
Comment (nicolas-cellier-aka-nice) [inline build.win32x86/common/Makefile:263]: 👍 should probably be back-ported to win32/win64 squeak/newspeak so as to enable build via MSYS

**Review by nicolas-cellier-aka-nice — APPROVED** [review]
Look OK from my eyes

File: build.win32x86/common/Makefile (hunk 1 above)
Comment (estebanlm) [reply build.win32x86/common/Makefile:263]: is already there for all win32... it needs to be ported to win64 :)

**Comment by estebanlm** [comment]
I will accept because the problem has nothing to do (and it will fixed with lib caching feature)

**Pull request merged**